docs/contributing.md

```diff
@@ -6,10 +6,15 @@ permalink: /contributing/
 
 We encourage you to participate in this open source project. We love pull requests, bug reports, ideas, (security) code reviews or any kind of positive contribution.
 
-Before you attempt to make a contribution please read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
+Thank you for taking the time to contribute to one of Mozilla's Android projects!
+
+Before contributing, please review our [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
 
 ## Docs
 
+* [https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING.md](https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING.md)
```

Comment:
Should be we have a title instead of the whole URL?
To be more consistent because in the next reference we title.

Reply:
Oops. Yes. That was a popy and caste error :)